### PR TITLE
Set up database with 101 objects of reviews. Server, client, webpack,…

### PR DESCRIPTION
**Pull request by chandra-nguyen**
… and babel are all set up and working together.

### CODE REVIEW

**Review by brennancaldwell — COMMENTED** [review]
Great work! Love the breadth of information you're handling in your DB. We ought to confer about ports: we should each be listening on different ports for the sake of combining our services later on, but that's an easy fix.

File: database/seeder.js [hunk 1]
```diff
@@ -1,36 +1,20 @@
...
-  communication: Number,
-  checkin: Number,
-  accuracy: Number,
-  location: Number,
-  value: Number
-});
-//create model for schema inputs
-const Review = mongoose.model('Review', reviewSchema);
 //seed database for 100 random entries
 for (var i = 0; i < 101; i++) {
```
Comment (Mongoos) [inline database/seeder.js:6]: It's really nitpicky, but this technically does 101 fake entries

File: server/index.js [hunk 2]
```diff
@@ -1,9 +1,57 @@
...
+    cleanliness: [],
+    accuracy: [],
+    communication: [],
+    checkin: [],
+    location: [],
+    value: []
+  };
+
+  return new Promise((resolve, reject) => {
+    Review.find({}, { cleanliness: 1, _id: 0 })
```
Comment (Mongoos) [inline server/index.js:20]: Sorry if I just don't understand mongoose as well, but do all these queries need to be separate or can they be bunched into one?

Comment (Mongoos) [reply server/index.js:20]: Other than that I think it looks great.  If youre getting a response from the server I say its approved!

**Review by Mongoos — APPROVED** [review]

**Pull request merged**